scripts/seed.py

```diff
@@ -0,0 +1,210 @@
+# Command to run : python manage.py shell --settings=settings.dev  < scripts/seed.py
+
+from django.conf import settings
+from django.core.files.uploadedfile import SimpleUploadedFile
+from django.contrib.auth.models import User
+from django.utils import timezone
+
+from allauth.account.models import EmailAddress
```

Comment:
Can you order the import statements lexicographically. It actually makes it more readable. Also group import statements like first python imports, then django imports, then django third party imports and then project specific imports.